kong/runloop/handler.lua

```diff
@@ -379,56 +532,50 @@ do
   end
 
 
-  update_plugins_iterator = function()
+  update_plugins_iterator = function(check_for_versions)
     local version, err = kong.cache:get("plugins_iterator:version", TTL_ZERO, utils.uuid)
```

Comment:
It seems that versions are checked still here even when adding `check_for_versions`. Should we skip checking versions here too if requested?

Reply:
Perhaps `check_for_versions` is not a good name.

What it means is: *while actually doing the build* (while making the `db.plugins:each(1000)`, or with routes), check from time to time if the version has changed.

Usually you want to do it but there are cases where you don't want to - when initializing stuff in declarative for example.